internal/crypto/ptls/ptls.go

```diff
@@ -19,7 +19,7 @@ import (
 	"k8s.io/client-go/transport"
 )
 
-// TODO decide if we need to expose the four TLS levels (secure, default, default-ldap, legacy) as config.
+// TODO decide if we need to expose the three TLS levels (secure, default, default-ldap) as config.
```

Comment:
I think we can delete this TODO. We decided not to make it configurable. We can always change our minds some day if we choose.

Reply:
deleted with next push